### PR TITLE
bitnami/dataplatform-bp2 Updated the resources to the Elasticsearch and logstash and README file is updated with Wavefront configuration for existing setup.

### PR DESCRIPTION
**Pull request by banicr**
**Description of the change**

Bitnami/dataplatform-bp2 Updated the resources to the Elasticsearch and logstash and README file is updated with Wavefront configuration for existing setup.

**Benefits**

Resources will get used efficiently and If you want to use an existing Wavefront deployment, we can edit the Wavefront Collector ConfigMap and add the blueprint configuration snippet under Prometheus sources.

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

**Review by miguelaeh — APPROVED** [review]
Thank you very much for the PR!!
LGTM!

**Comment by miguelaeh** [comment]
Please take a look to the failing job about the README generator before merging

File: bitnami/dataplatform-bp2/values.yaml [hunk 1]
```diff
@@ -481,7 +481,15 @@ logstash:
                 values:
                   - '{{ .Release.Name }}'
           topologyKey: "kubernetes.io/hostname"
-
+  ## @param extraEnvVars Array containing extra env vars to configure Logstash
```
Comment (miguelaeh) [inline bitnami/dataplatform-bp2/values.yaml:484]: This should fix the CI
```suggestion
  ## @param logstash.extraEnvVars Array containing extra env vars to configure Logstash
```

Comment (banicr) [reply bitnami/dataplatform-bp2/values.yaml]: Hi @miguelaeh, Thanks for the suggestion. Resolved this.

**Review by miguelaeh — APPROVED** [review]
LGTM!

**Pull request merged**